ManatiCR/FDI
# FDI-10: Bloques de contenido

[FDI-10](https://manati.atlassian.net/browse/FDI-10)
-
The default content is created when running the install so run:
`./scripts/fdi_local_install.sh` that command now also download and update the configuration translation

**Steps to test**
- Go to add or edit a node bloque de contenido the field image should be displayed only if the field tipo has the following values selected: Homepage: `Derechos`, `Homepage: FDI` or `Conocé tus derechos: Header`
- Create an administrador de contenido user
- Log in as the new user go to `/content` you should be able only to edit the nodes existent
- The field tipo should be deactivated for that role
- Ensure you can edit the content
---
- Check the spanish translations is working as expected

**Note:** It is not possible to export the images with the default nodes, should be added later.
